ceph/ceph_admin/bootstrap.py

```diff
@@ -273,12 +274,16 @@ def bootstrap(self: CephAdmProtocol, config: Dict):
         # if they are already not present in the default path
         copy_ceph_configuration_files(self, args)
 
-        # The provided image is used by Grafana service only when
-        # --skip-monitoring-stack is set to True during bootstrap.
-        if self.config.get("grafana_image"):
-            cmd = "cephadm shell --"
-            cmd += " ceph config set mgr mgr/cephadm/container_image_grafana"
-            cmd += f" {self.config['grafana_image']}"
-            self.installer.exec_command(sudo=True, cmd=cmd)
+        # Check for image overrides
+        if self.config.get("overrides"):
+            override_dict = dict(item.split("=") for item in self.config["overrides"])
+            supported_overrides = ["grafana", "keepalived", "haproxy"]
```

Comment:
can we could include other monitoring services as well, 
**Ref:** https://docs.ceph.com/en/latest/cephadm/services/monitoring/#using-custom-images

Reply:
Thanks... done